frontend/src/metabase/search/components/SearchSidebar/DropdownSidebarFilter/DropdownSidebarFilter.unit.spec.tsx

```diff
@@ -15,7 +15,7 @@ const mockFilter: SearchFilterComponent = {
       {!value || value.length === 0 ? "Display" : value}
     </div>
   ),
-  ContentComponent: ({ value, onChange, onApply }) => (
+  ContentComponent: ({ value, onChange }) => (
```

Comment:
I made the components that correspond to `ContentComponent` uncontrolled (which is what it should be) so we don't have to worry about "changing vs applying" semantics - when we decide that it's time to change the filter, we should just use `onChange` in `ContentComponent`